content/en/docs/tasks/debug-application-cluster/inside-container-developing.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Develop and debug the services in the container
+content_type: task
+---
+
+Kubernetes applications are generally composed of many microservices. Due to environment differences and resource constraints, these microservices somtimes can not run locally as source code. Therefore, developers need to rebuild the image, modify the image version of the workload and then wait for the Pod scheduling after coding. This causes the low efficiency of development in Kubernetes emvironment.
+
```

Comment:
```suggestion

This page describes how you can external tools to debug an application that's running in your cluster.
The tools covered are:
{{% thirdparty-content %}}

- [Nocalhost](https://nocalhost.dev)

```

It would be helpful to make a similar change to the Telepresence task, so that they match each other.